test/trivial/shannon/readWriteComplexTuple.stdin

```diff
@@ -1,2 +0,0 @@
-(9.8, 7.6)
```

Comment:
Yes.  It was the first thing I put on my misc branch with the idea of collecting a few other small unrelated things.  It's just that the next five unrelated things turned out to be related.

I could restructure it if you'd prefer.